tracker_git_preflight_integration_test.go

```
@@ -0,0 +1,125 @@
+		}
+	}
+}
+
+// TestIntegration_Preflight_GitInitClears confirms the same fixture proceeds
+// past preflight once `git init` has been run. Downstream NewEngine may still
+// fail for unrelated reasons (no API key) — the assertion is only that
+// preflight is no longer the failure point.
+func TestIntegration_Preflight_GitInitClears(t *testing.T) {
+	requireGit(t)
+	dir := t.TempDir()
+	cmd := exec.Command("git", "init", "-q")
```

golangci-lint (2.12.2)

[error] os/exec.Command must not be called. use os/exec.CommandContext

(noctx)